docs/source/user/cfunc.rst

```diff
@@ -116,13 +116,91 @@ assumed to be laid out in C order.  If the data is laid out in Fortran order,
 :func:`numba.farray` should be used instead.
 
 
+Dealing with C structures
+=========================
+
+
+With CFFI
+---------
+
+For applications that has a lot of states, it is useful to pass data in C
+structures.  To simplify the interop with C code, numba can convert a ``cffi``
```

Comment:
s/interop/interoperability/ (can't use the suggest edit thing here as it'd run past 80 chars)

Reply:
fixed